controllers/grid/CustomLocaleGridHandler.inc.php

```diff
@@ -241,8 +245,10 @@ function index($args, $request) {
 	 */
 	function getFilterForm() {
 		$customLocalePlugin = self::$plugin;
-		$templatePath = $customLocalePlugin->getTemplatePath();
-		return $templatePath . 'customLocaleGridFilter.tpl';
+		/* UZH CHANGE OJS-74 2019/03/05/mb adapt to OJS 3.1.2 */
+		/* $templatePath = $customLocalePlugin->getTemplatePath(); */
+		/* return $templatePath . 'customLocaleGridFilter.tpl'; */
+		return $customLocalePlugin->getTemplateResource('customLocaleGridFilter.tpl');
```

Comment:
I've split the template adaptations for the `stable-3_1_2` (just created) and `master` branches into a separate commit: https://github.com/asmecher/customLocale/commit/bca9b8a46099791360d54c18a59cf26331d25582